stetho-repl-js/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_NAME=Stetho REPL JavaScript module
+POM_ARTIFACT_ID=stetho-repl-js
```

Comment:
Do we maybe want to call this stetho-rhino or stetho-repl-rhino?  Using `js` here might limit us in the future (for instance, Nashorn and then even more broadly JSR-223).